Blazor_Foundation_6_Test_Shared/Documentation/Navigation/MenuDrilldown.razor

```diff
@@ -1,6 +1,152 @@
+        <OpenCodeDev.Blazor.Foundation.Components.Navigation.MenuDrilldown Vertical="true" data-auto-height="true" data-animate-height="true">
+            <li>
+                <NavLink href="#">Item 1</NavLink>
+                <MenuNested Vertical="true">
+                    <li><NavLink href="#">Item 1A</NavLink></li>
+                    <li><NavLink href="#">Item 1B</NavLink></li>
+                    <li><NavLink href="#">Item 1C</NavLink></li>
+                    <li><NavLink href="#">Item 1D</NavLink></li>
+                </MenuNested>
+            </li>
+            <li>
+                <NavLink href="#">Item 2</NavLink>
+                <MenuNested Vertical="true">
+                    <li><NavLink href="#">Item 2A</NavLink></li>
+                    <li><NavLink href="#">Item 2B</NavLink></li>
+                    <li><NavLink href="#">Item 2C</NavLink></li>
+                    <li><NavLink href="#">Item 2D</NavLink></li>
+                </MenuNested>
+            </li>
+            <li>
+                <NavLink href="#">Item 3</NavLink>
+                <MenuNested Vertical="true">
+                    <li><NavLink href="#">Item 3A</NavLink></li>
+                    <li><NavLink href="#">Item 3B</NavLink></li>
+                    <li><NavLink href="#">Item 3C</NavLink></li>
+                    <li>
+                        <NavLink href="#">Item 3D</NavLink>
+                        <MenuNested Vertical="true">
+                            <li><NavLink href="#">Item 3DA</NavLink></li>
+                            <li><NavLink href="#">Item 3DB</NavLink></li>
+                            <li><NavLink href="#">Item 3DC</NavLink></li>
+                            <li><NavLink href="#">Item 3DD</NavLink></li>
+                        </MenuNested>
+                    </li>
+                </MenuNested>
+            </li>
+            <li><NavLink href="#">Item 4</NavLink></li>
+            <li><NavLink href="#">Item 5</NavLink></li>
+        </OpenCodeDev.Blazor.Foundation.Components.Navigation.MenuDrilldown>
+        <hr />
+    </SectionLeft>
+</SectionWrapper>
+
+<SectionWrapper>
```

Comment:
Same you dont follow foundation's example it seems you didn't test that!!! its not a robot's job